From foo@baz Wed Jun  3 10:14:19 PDT 2009
Date: Wed, 03 Jun 2009 10:14:19 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: pata_rdc: fix build warnings

struct ata_port_info shouldn't be const, so remove that which fixes up
the compiler warnings.

Cc: Kevin Huang  <Kevin.Huang@rdc.com.tw>
Cc: Tomy Wang <Tomy.Wang@rdc.com.tw>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/pata_rdc/pata_rdc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/pata_rdc/pata_rdc.c
+++ b/drivers/staging/pata_rdc/pata_rdc.c
@@ -942,7 +942,7 @@ static struct scsi_host_template rdc_pat
 	ATA_BMDMA_SHT(DRIVER_NAME),
 };
 
-static const struct ata_port_operations rdc_pata_ops = {
+static struct ata_port_operations rdc_pata_ops = {
 	.inherits	= &ata_bmdma_port_ops,
 
 	.port_start	= rdc_pata_port_start,
